.circleci/regenerate.py

```diff
@@ -184,7 +185,7 @@ def unittest_workflows(indentation=6):
     env = jinja2.Environment(
         loader=jinja2.FileSystemLoader(d),
         lstrip_blocks=True,
-        autoescape=False,
+        autoescape=select_autoescape(enabled_extensions=('html', 'xml')),
```

Comment:
Not that there is anything bad with this change, but `.circleci` folder is not redistributed with the product